x-pack/qa/rolling-upgrade-basic/src/test/java/org/elasticsearch/upgrades/BasicLicenseUpgradeIT.java

```diff
@@ -24,20 +25,30 @@ public void testNewClusterHasActiveNonExpiringBasic() throws Exception {
 
     @SuppressWarnings("unchecked")
     private void checkBasicLicense() throws Exception {
-        Response licenseResponse = client().performRequest(new Request("GET", "/_license"));
-        Map<String, Object> licenseResponseMap = entityAsMap(licenseResponse);
-        Map<String, Object> licenseMap = (Map<String, Object>) licenseResponseMap.get("license");
-        assertEquals("basic", licenseMap.get("type"));
-        assertEquals("active", licenseMap.get("status"));
+        try {
```

Comment:
Does this work as an alternative? This way it only retries on a 404:

```
        final Request request = new Request("GET", "/_license");
        request.addParameter("ignore", "404");
        Response licenseResponse = client().performRequest(request);
        assertOK(licenseResponse);
```

Reply:
I didn't know about the `ignore` parameter, that looks much better 👍

Reply:
Full disclosure: nor did I until just now :grin: